fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+{% extends "base.html" %}
+
+{% block extra_headers %}
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no" />
+{% endblock %}
+
+{% block site_js %}
+  {% javascript 'android_feedback' %}
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'android-feedback' %}
+{% endblock %}
+
+{% block page_title %}{{ _('Firefox Feedback') }}{% endblock %}
```

Comment:
You should throw a Jinja2 comment before this line explaining the l10n situation in this template. Something along these lines:

```
{#
 # Note on L10N: In order to reduce the work required by translators for us to get this 
 # first phase done, we've copied the strings from Firefox for Fennec exactly. We should
 # fix the strings at some point after this is in production. See bug #xxxxxxx.
 #}